docs/source/reference/samplers/index.rst

```diff
@@ -11,6 +11,60 @@ The :mod:`~optuna.samplers` module defines a base class for parameter sampling a
 .. seealso::
     :ref:`user_defined_sampler` tutorial could be helpful if you want to implement your own sampler classes.
 
++----------------------------------+---------------+--------------+-------------+--------------+----------------------------------------+------------+----------------+
+|                                  | RandomSampler | GridSampler  | TPESampler  | CmaEsSampler |              NSGAIISampler             | QMCSampler | BoTorchSampler |
```

Comment:
Should we add SkoptSampler here?

Reply:
I think `SkoptSampler` should not be used since it is very slow. We can ignore it in this table.

Reply:
Sounds reasonable. Thank you!